src/app/session/start/page.tsx

```
@@ -0,0 +1,370 @@
+    isLoading: scheduledLoading,
+    error: scheduledError,
+  } = useScheduledSessions();
+  const [searchTerm, setSearchTerm] = useState('');
+
+  // State for schedule modal
+  const [isScheduleModalOpen, setIsScheduleModalOpen] = useState(false);
+  const [selectedTemplateId, setSelectedTemplateId] = useState<string | null>(
+    null,
+  );
+  const [selectedTemplateName, setSelectedTemplateName] = useState('');
+  const [selectedTemplate, setSelectedTemplate] =
```

ESLint

[error] 'selectedTemplate' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)